docs/multicluster/user-guide.md

```diff
@@ -117,19 +117,14 @@ To deploy Multi-cluster Controller in a dual-role cluster, please refer to
 
 #### Deploy in a Dedicated Leader Cluster
 
-1. Run the following command to import Multi-cluster CRDs in the leader cluster:
-
-   ```bash
-   kubectl apply -f https://github.com/antrea-io/antrea/releases/download/$TAG/antrea-multicluster-leader-global.yml
-   ```
-
-2. Install Multi-cluster Controller in the leader cluster. Since Multi-cluster
-   Controller runs as a namespaced Deployment, you should create the Namespace
-   first, and then apply the deployment manifest with the Namespace.
+Run the following command to import Multi-cluster CRDs and install Multi-cluster
+Controller in the leader cluster. Since Multi-cluster Controller runs as a namespaced
```

Comment:
"Since Multi-cluster Controller runs as..."

Let us rephrase it, like "Multi-cluster Controller is deployed into a Namespace. You must create the Namespace first...".

Reply:
Updated.